graphix/pattern.py

```diff
@@ -345,8 +356,43 @@ def standardize(self, method="local"):
             self._move_N_to_left()
             self._move_byproduct_to_right()
             self._move_E_after_N()
-        else:
-            raise ValueError("Invalid method")
+
+    def standardize_direct(self) -> None:
+        N_list = []
```

Comment:
Please use PEP8 naming (I'm planning to submit a through PR for that later).

Reply:
Fixed in